internal/setup/setup.go

```diff
@@ -410,27 +430,52 @@ func (yp *yamlPath) SameContent(other *yamlPath) bool {
 }
 
 type yamlArch struct {
-	list []string
+	List []string
```

Comment:
Let's not make this attribute public and instead keep it private because it is not needed.

Reply:
Per offline discussions, leaving it exported to avoid using the `yaml.IsZeroer` interface for `yamlArch`.